hugo.toml

```diff
@@ -184,11 +184,17 @@ js = [
 ]
 
 [[params.versions]]
-version = "v1.30"
-githubbranch = "v1.30.0"
+version = "v1.31"
+githubbranch = "v1.31.0"
 docsbranch = "main"
 url = "https://kubernetes.io"
 
+[[params.versions]]
+version = "v1.30"
+githubbranch = "v1.30.0"
```

Comment:
💭 Historically, we've also updated the `githbubarach` configuration value in all versions to point to the latest available patch version _(1.30.3, 1.29.7, 1.28.12, 1.27.16)_. Do we want to follow the same approach here?

Reply:
Happy for SIG Release (release docs team) to retain that responsibility, and to do it closer to go live.